pkgs/stdenv/linux/default.nix

```diff
@@ -188,6 +190,17 @@ in
       };
       coreutils = bootstrapTools;
       gnugrep = bootstrapTools;
+
+      fetchpatch = import ../../build-support/fetchpatch/default.nix {
+        inherit lib;
+        buildPackages.patchutils_0_3_3 = prevStage.patchutils_0_3_3.override {
+          forStdenvBootstrap = true;
+          # the parts of patchutils used by fetchpatch do not
+          # require perl, so we could eliminate this by patching
+          # patchutils' build scripts...
+          perl = bootstrapTools;
```

Comment:
I am not sure if bootstrapTools contains anything related to perl. If not then I think it would be cleaner to exclude it like makeWrapper.

Reply:
The `bootstrapTools` contain a copy of perl.  Mainly because autoconf/automake need it.

I would love to eliminate `perl` from the `bootstrapTools`.  I tried a while back and while it is probably possible it is a very major undertaking.  If it ever were eliminated we could (as my comment explains) apply a small patch to `patchutils` to drop all the parts of it that need perl (`fetchpatch` doesn't use those parts).